plugins.d/pip.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+echo "🐍 Python pip"
```

Comment:
Since `pip freeze` doesn't list `pip` itself it's necessary to first call `pip install -U pip` to self update.